### DISCUSSION TITLE
review

### DISCUSSION THREAD

**Discussion by YoussefBechara** [Soulter/hugging-chat-api#153] [General]
Thank you soooooooooooooooooooo much, god bless you and your work you saved my ass, i was scraping huggingchat with selenium and it keptmgiving me bs errors, agaain THANKSSSSS A LOTTTT

**Comment by mrTommorow** [comment]
would be nice if you could choose assistants by the url in there profile as my script is struggling to find assistants by name, specifically names with spaces in them.

**Reply by Soulter** [reply]
Hey, you can add your assistant directly by the assistant id. Here are some codes may help you:
```py
YOUR_ASSISTANT_ID = "xxxxxx"
chatbot.new_conversation(assistant=YOUR_ASSISTANT_ID, switch_to = True) # switch to the new conversation
```
